bookserver/schemas.py

```diff
@@ -154,3 +155,24 @@ class LogRunIncoming(BaseModelNone):
     suffix: Optional[str]
     partner: Optional[str]
     sid: Optional[str]
+
+
+# Schemas for Completion Data
+# ---------------------------
+class LastPageDataIncoming(BaseModel):
+    last_page_url: str  # = Field(None, alias="lastPageUrl") is the manual way
+    course_id: str = Field(None, alias="course")
```

Comment:
Is this something we could instead change in the JS to make it consistent, instead of working around it here?